### ISSUE TITLE
How to get better results

### ISSUE THREAD

**Issue by kavanalp** [YiyanXu/DiffRec#13]
I used the default hyper parameters "!python main.py --cuda --dataset=ml-1m_clean --data_path=../datasets/ml-1m_clean/"
the results are less than 0.1 and the loss is about 180

**Comment by YiyanXu** [comment]
To reproduce the results and perform fine-tuning of the hyperparameters, please refer to lines 138-139 in the **inference.py** file. Here, you will find the specific hyperparameter settings.

**Comment by kavanalp** [comment]
@YiyanXu  Thank you for you answer!
I have tried Diifrec and L_Diffrec with the hyperparameters you provided in lines 138-139 of the inference.py file, but the results are still not satisfactory.
`!python main.py --cuda --dataset=ml-1m_clean --data_path=../datasets/ml-1m_clean/  --emb_path=../datasets/ --lr1=0.001 --lr2=0.0005 --wd1=0.0 --wd2=0.0 --batch_size=400 --n_cate=2 --in_dims=[300] --out_dims=[] --lamda=0.03 --mlp_dims=[300] --emb_size=10 --mean_type='x0' --steps=100 --noise_scale=0.005 --noise_min=0.005 --noise_max=0.02 --sampling_steps=0 --reweight=1 --log_name=_AE.pth --gpu='1'`

Best Epoch 075 
[Valid]: Precision: 0.0648-0.0606-0.0536-0.0465 Recall: 0.061-0.1098-0.2234-0.3567 NDCG: 0.0786-0.0931-0.132-0.1789 MRR: 0.1533-0.1653-0.1726-0.1742
[Test]: Precision: 0.0546-0.0506-0.0419-0.0344 Recall: 0.0981-0.173-0.3187-0.4756 NDCG: 0.0825-0.1085-0.1573-0.2052 MRR: 0.1368-0.1494-0.1567-0.1585
End time:  2023-12-29 08:41:05
